adagucserverEC/CDataReader.cpp

```diff
@@ -1018,12 +1009,15 @@ int CDataReader::open(CDataSource *dataSource, int mode, int x, int y, int *grid
 #endif
   }
 
-  /*
-   * DataPostProc: Here our datapostprocessor comes into action! It needs scale and offset from datasource.
-   * This is stage1, only AX+B will be applied to scale and offset parameters
-   */
-  CDataPostProcessor::getCDPPExecutor()->executeProcessors(dataSource, CDATAPOSTPROCESSOR_RUNBEFOREREADING);
-
+  if (enablePostProcessors) {
+    /*
+     * DataPostProc: Here our datapostprocessor comes into action! It needs scale and offset from datasource.
+     * This is stage1, only AX+B will be applied to scale and offset parameters
```

Comment:
Is this comment still correct, or has postprocessing been extended beyond this?

Reply:
Yes, scale and offset parameters are used in the reader to apply ax+b formula directly.

Reply:
Removed comments